src/components/shared/form/createProject.tsx

```diff
@@ -0,0 +1,291 @@
+            hasError={formErrors.includes('pName')}
+          />
+          {formErrors.includes('pName') && (
+            <ErrorMessage value="Project Name" />
+          )}
+          <FormHint>
+            Make your project name simple, specific and memorable
+          </FormHint>
+          ​<FormLabel htmlFor="description">Description</FormLabel>
+          <FormTextArea
+            name="pDesc"
+            onChange={handleChange}
+            onBlur={handleBlur}
+            value={formInputs['pDesc'].val}
+            displayCharCount={true}
+            maxCharCount={135}
+            hasError={formErrors.includes('pDesc')}
+          />
+          {formErrors.includes('pDesc') && (
+            <ErrorMessage value="Project Description" />
+          )}
+          <FormHint>Describe your project in a single tweet</FormHint>​
+          <FormLabel htmlFor="project-type">Project Type</FormLabel>
+          <FormSelectInput
+            name="pType"
+            options={projectTypes}
+            onChange={handleChange}
+            onBlur={handleBlur}
+            hasError={formErrors.includes('pType')}
+            placeholder="Select a Project Type"
+          />
+          {formErrors.includes('pType') && (
+            <ErrorMessage value="Project Type" />
+          )}
+          <FormHint>What category does your project belong to?</FormHint>​
+          <FormLabel htmlFor="project-repo">Project Repo</FormLabel>
+          <FormInput
+            name="pRepo"
+            type="text"
+            onChange={handleChange}
+            onBlur={handleBlur}
+            hasError={formErrors.includes('pRepo')}
+          />
+          {formErrors.includes('pRepo') && (
+            <ErrorMessage value="Project Repo" />
+          )}
+          <FormHint>Share your project repo (GitHub, GitLib etc)</FormHint>​
+          <FormLabel htmlFor="launch-date">Launch Date</FormLabel>
+          <FormInput
+            name="pLaunch"
+            type="date"
+            onChange={handleChange}
+            onBlur={handleBlur}
+            hasError={formErrors.includes('pLaunch')}
+          />
+          {formErrors.includes('pLaunch') && (
+            <ErrorMessage value="Project Launch" />
+          )}
+          <FormHint>
+            Keep you and your team accountable with a launch date
+          </FormHint>
+          ​
+          <FormLabel htmlFor="communication-platform">
+            Communication Platform
+          </FormLabel>
+          <FormInput
+            name="pComm"
+            type="text"
+            onChange={handleChange}
+            onBlur={handleBlur}
+            hasError={formErrors.includes('pComm')}
+          />
+          {formErrors.includes('pComm') && (
+            <ErrorMessage value="Communication Platform" />
+          )}
+          <FormHint>
+            Where will you communicate? Share the invite link to your workspace
+            (Slack, Discord, Gitter etc)
```

Comment:
We'll support Slack or Discord to begin with so we can change this.
```suggestion
            (Slack or Discord)
```

Reply:
fixed